docs/src/components/features/panel/panel.html

```diff
@@ -1,35 +1,37 @@
-<article class="panel">
-  <h1 class="panel-title">Panel title</h1>
-  <div class="panel-rows">
-    <section class="panel-group">
-      <header class="panel-header">
-        <h2>Group header</h2>
-        <span>Secondary information</span>
+<section class="panel">
+    <header class="panel-header row-center-x">
```

Comment:
Can we remove `row-center-x`? Doing so could move this out of prototype. Here and elsewhere.